### PR TITLE
PVC strategies and Without Keycloak content

### PR DESCRIPTION
**Pull request by MichalMaler**
Signed-off-by: Michal Maléř <mmaler@redhat.com>

* Adding new content
* Restructuring and rewriting the old one
* Adding a new attribute

### CODE REVIEW

File: src/main/pages/che-7/administration-guide/proc_configuring-prod-short-to-work-without-identity-provider.adoc [hunk 1]
```diff
@@ -0,0 +1,67 @@
+// Module included in the following assemblies:
+//
+// configuring-authorization
+
+[id="configuring-prod-short-to-work-without-identity-provider_{context}"]
+= Configuring {prod-short} to work without {identity-provider}
+
+By default, {prod-short} is deployed in multi-user mode where {identity-provider} and PostgreSQL are enabled by default. However, it is possible to deploy it in the single-user mode too, by changing the `CHE_MULTIUSER` value to `false`. In that case, neither {identity-provider} nor PostgreSQL is installed.
```
Comment (yhontyk) [inline src/main/pages/che-7/administration-guide/proc_configuring-prod-short-to-work-without-identity-provider.adoc]:
```suggestion
By default, {prod-short} is deployed in multiuser mode where {identity-provider} and PostgreSQL are enabled by default. However, it is possible to deploy it in the single-user mode too, by changing the `CHE_MULTIUSER` value to `false`. In that case, neither {identity-provider} nor PostgreSQL is installed.
```

File: src/main/pages/che-7/administration-guide/proc_configuring-prod-short-to-work-without-identity-provider.adoc [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+. Install {prod-short} in the single-user mode:
++
+[subs="+quotes,+attributes"]
+----
+$ {prod-cli} server:start --platform __<platform>__ --installer operator  --che-operator-cr-patch-yaml cr-patch.yaml
+----
++
+Depending on the strategy used, replace the `_<platform>_` option in the above example with `crc`, `minishift`, or `openshift`. 
+ifeval::["{project-context}" == "che"]
+Available are also Kubernetes- native platforms such as: `minikube`, `microk8s`, `k8s`, and `docker-desktop`
```
Comment (yhontyk) [inline src/main/pages/che-7/administration-guide/proc_configuring-prod-short-to-work-without-identity-provider.adoc]:
```suggestion
Kubernetes-native platforms such as `minikube`, `microk8s`, `k8s`, and `docker-desktop` are also available.
```

Comment (MichalMaler) [reply src/main/pages/che-7/administration-guide/proc_configuring-prod-short-to-work-without-identity-provider.adoc]: Thanks, Yana, applying.

**Review by yhontyk — APPROVED** [review]

File: src/main/pages/che-7/overview/proc_configuring-a-che-workspace-with-a-persistent-volume-strategy.adoc [hunk 3]
```diff
@@ -0,0 +1,137 @@
...
+== Configuring a PVC strategy strategy by editing a configMap
+
+Based on the {prod-short} installation method, configMaps can be used to customize the working environment. A configMap is provided as an editable file that lists options to customize the {prod-short} environment. This method of configuring a persistent volume claim (PVC) strategy for a {prod-short} workspace is available only for the Helm installation.
+
+Changes to a configMap created during Operator installation are not permanent because the Operator overwrites them back to default.
+
+.Prerequisites
+
+* {prod-short} is installed using the Helm Chart.
+* A Kubernetes orchestration tool, the OpenShift command-line tool, `oc`, or the Kubernetes clusters controlling command-line tool, `kubectl`, is installed.
```
Comment (yhontyk) [inline src/main/pages/che-7/overview/proc_configuring-a-che-workspace-with-a-persistent-volume-strategy.adoc:61]: Should we maybe use the attribute `orch-cli` in sentences like these?

**Pull request merged**